packages/jaeger-ui/src/components/SearchTracePage/SearchForm.jsx

```diff
@@ -287,15 +283,18 @@ export class SearchFormImpl extends React.PureComponent {
         >
           <Field
             name="service"
-            component={AdaptedVirtualSelect}
+            component={AdaptedSelect}
```

Comment:
any idea why Service / Operation dropdowns allow only 7 entries simultaneously visible, but Lookback allows 8?

Reply:
That's interesting, I'll check this out